workflows/nf_aggregate/main.nf

```diff
@@ -3,8 +3,12 @@
 //
 
 include { SEQERA_RUNS_DUMP     } from '../../modules/local/seqera_runs_dump'
+includeConfig '../../modules/local/seqera_runs_dump/nextflow.config'
 include { PLOT_RUN_GANTT       } from '../../modules/local/plot_run_gantt'
+includeConfig '../../modules/local/plot_run_gantt/nextflow.config'
 include { MULTIQC              } from '../../modules/nf-core/multiqc'
+includeConfig '../../modules/nf-core/multiqc/nextflow.config'
+
```

Comment:
Are you proposing this new syntax? It is not currently supported

Reply:
Hah, not intentionally